pygenometracks/tests/test_bed_and_gtf_tracks.py

```diff
@@ -98,3 +128,16 @@ def test_plot_tracks_bed_and_gtf():
     assert res is None, res
 
     os.remove(outfile.name)
+
+
+def test_plot_tracks_bed_with_maxLab():
+
+    outfile = NamedTemporaryFile(suffix='.png', prefix='pyGenomeTracks_test_', delete=False)
+    args = "--tracks {0}/bed_maxLab_tracks.ini --region X:2000000-3500000 --trackLabelFraction 0.2 --width 38 " \
+           "--dpi 130 --outFileName  {1}".format(ROOT, outfile.name).split()
+    pygenometracks.plotTracks.main(args)
+    print("saving test to {}".format(outfile.name))
```

Comment:
this can be removed I think

Reply:
You mean you would not do a test for this functionality so I remove the test, the ini file and the png?

Reply:
I wouldn't remove any test.  Let's see what Bjoern thinks, maybe he found it redundant with another test?

Reply:
Oh no tests are great, I love tests, but the print is not needed right?